virttest/qemu_vm.py

```diff
@@ -1669,8 +1687,23 @@ def sort_key(dev):
                 LOG.warn(txt)
 
         smp_err = ""
+        SMP_PREFER_VERSION_SCOPE = '[6.2.0, )'
+        #  In the calculation of omitted sockets/cores/threads: we prefer
+        #  sockets over cores over threads before 6.2, while preferring
+        #  cores over sockets over threads since 6.2.
```

Comment:
We should clear the machine type info here, the current comment is a little confuesing. Please check this.

Reply:
Thanks for the suggestion, but the machine type setting is our internal information. 
For upstream I think such information is sufficient.
And the interface is provided here for the downstream to choose at will.